i18n/en-US.yml

```diff
@@ -763,6 +763,7 @@ components:
     errorUpdatingProfile: Error updating profile.
     fieldUpdated: This setting has been updated.
     fields:
+      add-companion-form: Companions
```

Comment:
I think this should say 'Travel Companions' to match the header of that section.